Package.swift

```diff
@@ -180,6 +184,19 @@ let package = Package(
             path: "AdyenTwint",
             exclude: ["Info.plist"]
         ),
+        .target(
+            name: "AdyenCheckout",
+            dependencies: [
+                .target(name: "Adyen"),
```

Comment:
Would we remove these later from the top level libraries?

Reply:
Yes most modules won't be exposed as separate to outside, only usable between our modules